Fixed download links

dariox86 opened the pull request:
[sources.txt](https://github.com/george-hopkins/opentom/files/2209172/sources.txt)

jkoan committed:
No extras, only navit

jkoan committed:
Now only Navit

jkoan committed:
Add cacheing for ci

jkoan committed:
Fixed path

jkoan committed:
Remove something to test if cache is working

(must optimize cache bacause current cache will not update navit :D)

jkoan committed:
More optimisations

jkoan committed:
Ci...

jkoan committed:
if [];then;fi

jkoan committed:
Dont say anything :D

jkoan committed:
Add full bluez

jkoan committed:
Updated Badgets

jkoan committed:
Fix Spelling

jkoan committed:
Make updates...

jkoan committed:
hopefully this works

jkoan committed:
Update circle.yml

jkoan committed:
Better Output?

jkoan committed:
Update sources.txt

jkoan committed:
Update sources.txt

jkoan committed:
Update sources.txt

jkoan committed:
Update to libpng-1.6.29.tar.gz

jkoan committed:
First changes on README

jkoan committed:
Update README.md

jkoan committed:
Update to last working state

jkoan committed:
update circleci

jkoan committed:
update

jkoan committed:
update

jkoan committed:
install git

jkoan committed:
add -y

jkoan commented:
Hello @dariox86,
The Pull-request you started is from my work, which is:
1) Not Production ready
2) A system for TomTom devices with special scope on a navigation system (Navit), but without support to run other software (in core configuration)

But you are right that i also updated the sources. If this helps someone, i can do a PR for the sources here as well. @george-hopkins sounds that good for you?

dariox86 commented:
Feel free to update the sources as you wish.

bignaux committed:
Update HOWTO_install_tomplayer.txt

RichardH-code committed:
Support GO 710; support swap and minix partitions; fix suspend, bluetooth

jkoan committed:
Add build-essential

jkoan committed:
Add wget

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update sources.txt

jkoan committed:
Update sources.txt

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update config.yml

jkoan committed:
Update Patch

jkoan committed:
Update config.yml

jkoan committed:
Update sources.txt

jkoan committed:
Update nxlib and microwin links

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
Update Makefile

jkoan committed:
fix build because sources changed, and other stuff optimized

jkoan committed:
Merge branch 'master' of https://github.com/navit-gps/navittom

jkoan committed:
fix Download as  i had all file local oops :D

jkoan committed:
fix remaining Download instances

jkoan committed:
Add Dowloads to cache

jkoan committed:
move cache after checkout to be sure to have the last sources.txt

jkoan committed:
fix Macro as well

jkoan committed:
Fix applications as well

jkoan committed:
Rename Setup step

jkoan committed:
Add mkdir for logs as this is not generated when Downloads are cached

jkoan committed:
require logs dir to be created

lufv committed:
Update navit_opentom.patch

Old patch file didn't have new lines that have been added in tomtom.xslt update on November 8, 2019 (latest patch update was on April 3, 2019)
